proposals/target-typed-new.md

```diff
@@ -76,13 +68,12 @@ Action a = new(); // no constructor found
 
 ### Miscellaneous
 
-`throw new()` is allowed.
-
-Target-typed `new` is not allowed with binary operators.
-
-It is disallowed when there is no type to target: unary operators, collection of a `foreach`, in a `using`, in a deconstruction, in an `await` expression, as an anonymous type property (`new { Prop = new() }`), in a `lock` statement, in a `sizeof`, in a `fixed` statement, in a member access (`new().field`), in a dynamically dispatched operation (`someDynamic.Method(new())`), in a LINQ query, as the operand of the `is` operator, as the left operand of the `??` operator,  ...
+The following are consequences of the specification:
 
-It is also disallowed as a `ref`.
+- `throw new()` is allowed (the target type is `System.Exception`)
+- Target-typed `new` is not allowed with binary operators.
+- It is disallowed when there is no type to target: unary operators, collection of a `foreach`, in a `using`, in a deconstruction, in an `await` expression, as an anonymous type property (`new { Prop = new() }`), in a `lock` statement, in a `sizeof`, in a `fixed` statement, in a member access (`new().field`), in a dynamically dispatched operation (`someDynamic.Method(new())`), in a LINQ query, as the operand of the `is` operator, as the left operand of the `??` operator,  ...
```

Comment:
>as the operand of the `is` operator

:memo: This is only the left-hand-side operand of the `is` operator.

Reply:
Thinking more about this, we disallow `default` in patterns, particularly to be not confused with `default:`, but perhaps it would be more consistent to disallow all typeless expressions.